### DISCUSSION TITLE
How to enable Terminal Access for project members only?

### DISCUSSION THREAD

**Discussion by gustavo1ms** [argoproj/argo-cd#10284] [Q&A] [answered]
Hi guys. 
How can I enable terminal access for project members only? 
I've a Argocd and I enabled the terminal access to pods in my projects, but each project has its own members and this members can only access their own project. These members are in the Developer group. 
I need members of developer group to be able to access your projects terminal.  The admin users are able to login normally. 

Can anyone help me with this question?

**Comment by crenshaw-dev** [comment, marked as answer]
Answer is here: https://github.com/argoproj/argo-cd/issues/10299

> @gustavo1ms this should be possible with a line something like this in the argocd-rbac-cm:
> 
>         p, developers, exec, create, my-project/*, allow
> 
> Docs for Web Terminal: https://argo-cd.readthedocs.io/en/stable/operator-manual/web_based_terminal/
